tests/python/relay/test_any.py

```diff
@@ -801,6 +801,27 @@ def test_any_ndarray_size():
     verify_any_ndarray_size((2, 2))
     verify_any_ndarray_size((1, 2, 3, 4))
 
+def test_any_consecutive_broadcast():
+    data0 = relay.var("data0", shape=any_dims(2), dtype='float32')
+    data1 = relay.var("data1", shape=any_dims(2), dtype='float32')
+    data2 = relay.var("data2", shape=any_dims(2), dtype='float32')
+    data3 = relay.var("data3", shape=any_dims(2), dtype='float32')
+
+    out0 = data0 + data1
+    out1 = data0 * data1
+    out2 = out0 - out1
+
+    out3 = data2 + data3
+    out4 = data2 * data3
+    out5 = out3 - out4
+
+    out6 = out2 * out5
+
+    mod = tvm.IRModule()
+    mod['main'] = relay.Function([data0, data1, data2, data3], out6)
+
+    relay.create_executor("vm", mod=mod, ctx=tvm.cpu(), target="llvm")
```

Comment:
let's use `check_results` and feed it with data so that we can run it on both gpu and cpu